codex/logutils.nim

```diff
@@ -0,0 +1,121 @@
+import std/options
+import std/sequtils
+import std/strutils
+import std/sugar
+import std/typetraits
+
+import pkg/chronicles except toJson, `%`
+# import pkg/faststreams
+from pkg/libp2p import Cid, MultiAddress, `$`
+import pkg/questionable
+import pkg/stew/byteutils
+import pkg/stint
+import pkg/upraises
+
+import ./utils/json
+
+export byteutils
+export chronicles except toJson, formatIt, `%`
+export questionable
+export sequtils
+export strutils
+export sugar
+export upraises
+export json
+
```

Comment:
One thing that'd be pretty useful here would be a module doc (`##`) explaining the same thing you did in the PR description. 

Just the rationale and the basics of using it for other devs to be able to see. Otherwise other's might not understand why this was done and just pull in chronicles again.

Reply:
Added.